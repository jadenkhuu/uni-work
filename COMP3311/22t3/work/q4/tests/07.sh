psql racing -c "select * from q4('trot') order by horse;"
