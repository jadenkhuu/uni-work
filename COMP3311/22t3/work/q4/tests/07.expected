     horse     | average 
---------------+---------
 Harry Trotter |   22555
 Hot to Trot   |   52777
 Leon Trotsky  |   29380
 Tater Trot    |   11133
(4 rows)
